src/main/java/net/sf/jabref/importer/fileformat/BibtexParser.java

```diff
@@ -224,10 +224,14 @@ private void parseAndAddEntry(String type) {
          * for the user.
          */
         try {
+            String commentsAndEntryTypeDefinitionString = dumpTextReadSoFarToString();
+
             BibEntry entry = parseEntry(type);
 
+            entry.setParsedSerialization(commentsAndEntryTypeDefinitionString+dumpTextReadSoFarToString());
+            entry.setCommentsBeforeEntry(commentsAndEntryTypeDefinitionString.substring(0,commentsAndEntryTypeDefinitionString.lastIndexOf('@')));
```

Comment:
It is not the `COMMENT_SYMBOL` but the general "start of bibtex entry/string/preamble/comment" symbol.
`@` as `char` is used in various places in the parser. However, a) I can't find a good name for the final field and b) I prefer touching this code as little as possible -> I stick to the `'@'` here :)